cern_access/indico_cern_access/templates/grant_access.html

```diff
@@ -6,7 +6,8 @@
         <p>
             {%- trans %}
                 All selected registrants provided the additional personal data needed to access the
-                CERN site. Granting them access will immediately send them their access tickets.
+                CERN site. Granting them access will immediately send them their access tickets,
+                if emailing tickets option is enabled.
```

Comment:
"if emailing tickets option" sounds wrong. let's put "if emailing tickets is enabled" or "if the 'NAME OF THE OPTION' option is enabled"